Interfaces/API/Job.py

```diff
@@ -578,39 +579,33 @@ def setDestination( self, destination ):
     """
     kwargs = {'destination':destination}
     if isinstance( destination, basestring ):
-      if not re.search( '^DIRAC.', destination ) and not destination.lower() == 'any':
-        result = self.__checkSiteIsValid( destination )
+      destination = [destination]
+      description = 'User specified destination site'
+    else:
+      description = 'List of sites selected by user'
+    if isinstance( destination, list ):
```

Comment:
Right, but if it is an int or a dict, this is a problem and should be spotted... I agree it is probably useless if the calling code is correct ;-)
Again this is how it was...